static/site.css

```diff
@@ -767,7 +762,7 @@ ul.guide li {
   justify-content: flex-start;
   padding-left: 14px;
 }
-@media (max-width: 799px) and (orientation: portrait) {
+@media (max-width: 799px) {
```

Comment:
prevents a bug where when `height < width < 799` the hamburger menu does not work

Reply:
https://github.com/gbtami/pychess-variants/assets/126312812/72696ff1-bb0e-451c-8609-ac7b8e3cf7f7